php-migrations/Emergence/People/20171014_phone-length.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+// skip conditions
+$skipped = true;
+if (!static::tableExists('people')) {
+    printf("Skipping migration because table `people` does not exist yet\n");
+    return static::STATUS_SKIPPED;
+}
+
+// migration
+if (static::getColumn('people', 'Phone')) {
+    print("Altering length of field `Phone` on table `people`'\n");
+    DB::nonQuery("ALTER TABLE `people` CHANGE COLUMN `Phone` `Phone` DECIMAL(15) UNSIGNED NULL DEFAULT NULL  COMMENT '' AFTER `Email`;");
+    $skipped = false;
+}
+
+if (!static::getColumn('history_people', 'CustomerEmailRedirect')) {
```

Comment:
What is `CustomerEmailRedirect` ?

Reply:
Grr I check these again and again and every time I mess up one of those getColumn fields b\c I copy it from a past migration. This one's on me ☹️